src/envoy/http/mixer/report_data.h

```diff
@@ -114,6 +119,30 @@ class ReportData : public ::istio::control::http::ReportData {
     return ExtractGrpcStatus(trailers_, status) ||
            ExtractGrpcStatus(headers_, status);
   }
+
+  // Get rbac permissive policy attributes.
+  void GetRBACPermissiveAttributes(std::string *resp_code,
+                                   std::string *policy_id) const override {
+    const auto filter_meta = info_.dynamicMetadata().filter_metadata();
+    const auto filter_it =
+        filter_meta.find(Extensions::HttpFilters::HttpFilterNames::get().Rbac);
+    if (filter_it == filter_meta.end()) {
+      return;
+    }
+
+    const auto &data_struct = filter_it->second;
```

Comment:
if tcp rbac needs these code,  move them into utils folder.

Reply:
erfer to move later(as mentioned in the other comment).